core/ui.js

```diff
@@ -606,14 +606,14 @@ function onNewMessageNodeAdded(messageNode)
 
     // check if there are any view-once messages for this message ID
     // open viewOnce indexedDB
-    saveViewOnceToIndexedDBIfNecessary(msgID, messageNode);
+    checkIfMessageIDIsInViewOnceDB(msgID, messageNode);
```

Comment:
Let's call this `restoreViewOnceMessageIfNeeded`, like `restoreDeletedMessageIfNeeded`

Reply:
Sorry I somehow misread your comment.